### PR TITLE
Integrate spdlogs

### PR DESCRIPTION
**Pull request by wp19991**

### CODE REVIEW

File: tests/logging_test.cc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    ASSERT_EQ(info_count, 1);
+    ASSERT_EQ(warn_count, 1);
+    file.close();
+    std::remove("log/test.log");
+}
+
+// Here tests trivial things for the logging
+//TEST(LoggingTest, TestLogTrivial) {
+//    metable::NodeID node_id;
+//    RAFTCPP_LOG(RLL_DEBUG) << node_id;
```
Comment (jovany-wang) [inline tests/logging_test.cc:43]:
```suggestion
//    METABLE_LOG(RLL_DEBUG) << node_id;
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -52,4 +52,17 @@ if(WIN32)
 endif()
 
 #METABLE_DEFINE_TEST(metable_server SRC_FILES src/server/server_main.cc LINKS gflags)
-METABLE_DEFINE_TEST(metable_server SRC_FILES src/server/server_main.cc)
+#METABLE_DEFINE_TEST(metable_server SRC_FILES src/server/server_main.cc LINKS common_lib)
```
Comment (jovany-wang) [inline CMakeLists.txt:55]: Why comment out this line? I believe we should produce the metrable_server binary.

**Review by jovany-wang — COMMENTED** [review]
Also the lint CI failed. You should format your code style.

File: tests/logging_test.cc (hunk 1 above)
Comment (wp19991) [reply tests/logging_test.cc:43]: ok

File: CMakeLists.txt (hunk 2 above)
Comment (wp19991) [reply CMakeLists.txt:55]: yes,you are right.I was wrong to think about it.

Comment (wp19991) [reply CMakeLists.txt:55]: You're right. I only noticed the modification of gtest. I didn't look back at this.

**Review by jovany-wang — APPROVED** [review]

**Pull request merged**